app/views/course/experience_points_records/_experience_points_record.html.slim

```diff
@@ -0,0 +1,4 @@
+= content_tag_for(:tr, experience_points_record)
+  td = experience_points_record.points_awarded
+  td = experience_points_record.reason
+  td = awarder_name(experience_points_record)
```

Comment:
`experience_points_record.creator` is a User. Should we fetch the CourseUser, or it doesn't matter?